worldedit-core/src/main/java/com/sk89q/worldedit/command/SelectionCommands.java

```diff
@@ -258,7 +258,7 @@ public void chunk(
                     .clampY(minChunkY, maxChunkY);
 
             min = minChunk.shl(CHUNK_SHIFTS, CHUNK_SHIFTS_Y, CHUNK_SHIFTS);
-            max = maxChunk.shl(CHUNK_SHIFTS, CHUNK_SHIFTS_Y, CHUNK_SHIFTS).add(15, world.getMaxY(), 15);
+            max = maxChunk.shl(CHUNK_SHIFTS, CHUNK_SHIFTS_Y, CHUNK_SHIFTS).add(15, 255, 15);
```

Comment:
Why was this changed back?

Reply:
The description of the command specifically states //chunk will select in a 256 block range, and upstream uses 255 currently.

Reply:
Interesting, but going by description of the command, you can already use these commands out of the typical range.